packages/react-instantsearch-hooks-dom/src/widgets/__tests__/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,375 @@
+        initialUiState={{
+          indexName: {
+            hierarchicalMenu: {
+              'hierarchicalCategories.lvl0': [
+                'Cameras & Camcorders > Digital Cameras',
+              ],
+            },
+          },
+        }}
+      >
+        <VirtualHierarchicalMenu attributes={hierarchicalAttributes} />
+        <Breadcrumb attributes={hierarchicalAttributes} />
+      </InstantSearchHooksTestWrapper>
+    );
+
+    await wait(0);
+
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div
+          class="ais-Breadcrumb"
+        >
+          <ul
+            class="ais-Breadcrumb-list"
+          >
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                Home
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                Cameras & Camcorders
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item ais-Breadcrumb-item--selected"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              Digital Cameras
+            </li>
+          </ul>
+        </div>
+      </div>
+    `);
+  });
+
+  test('transform the passed items', async () => {
+    const { searchClient, hierarchicalAttributes } =
+      getBreadcrumbSearchClient();
+
+    const { container } = render(
+      <InstantSearchHooksTestWrapper
+        searchClient={searchClient}
+        initialUiState={{
+          indexName: {
+            hierarchicalMenu: {
+              'hierarchicalCategories.lvl0': [
+                'Cameras & Camcorders > Digital Cameras',
+              ],
+            },
+          },
+        }}
+      >
+        <VirtualHierarchicalMenu attributes={hierarchicalAttributes} />
+        <Breadcrumb
+          attributes={hierarchicalAttributes}
+          transformItems={(items) =>
+            items.map((item) => ({ ...item, label: item.label.toUpperCase() }))
+          }
+        />
+      </InstantSearchHooksTestWrapper>
+    );
+
+    await wait(0);
+
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div
+          class="ais-Breadcrumb"
+        >
+          <ul
+            class="ais-Breadcrumb-list"
+          >
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                Home
```

Comment:
notable that the "home" item doesn't change, as it's not part of the items. Not really a problem though

Reply:
Yes, we'll have the `translations` prop to change this label.